tests/eeid_plugin/host/host.c

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+
+#if defined(_WIN32)
+#include <ShlObj.h>
+#include <Windows.h>
+#endif
+
+#include "../../../host/sgx/quote.h"
+#include "../test_helpers.h"
+#include "eeid_plugin_u.h"
+
+#define SKIP_RETURN_CODE 2
+
+int main(int argc, const char* argv[])
+{
+#ifdef OE_LINK_SGX_DCAP_QL
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+
+    if (argc != 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE\n", argv[0]);
+        exit(1);
+    }
+
+    // Skip test in simulation mode.
```

Comment:
Expand comment to say why this won't work in simulation mode?

Reply:
Memory alignment issues, the same as in `tests/attestation_plugin`. (Comment added.) Can probably be fixed by re-allocating heap pages in simulation mode, but I fear that might snowball into reallocating everything else too...